Backport of commit id 85fd0fe6fc002deba03fe36c1c9726cde1e6331c

i2c: piix4: Use different message for AMD Auxiliary SMBus Controller

Same messages for AMD main and auxiliary SMBus controllers lead to confusion,
this patch is to remove confusion and keep consistent with non-AMD products.

Signed-off-by: Shane Huang <shane.huang@amd.com>
Reviewed-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Wolfram Sang <wsa@the-dreams.de>
Signed-off-by: Arindam Nath <arindam.nath@amd.com>
diff -Naur a/drivers/i2c/busses/i2c-piix4.c b/drivers/i2c/busses/i2c-piix4.c
--- a/drivers/i2c/busses/i2c-piix4.c	2014-05-06 22:10:41.501656511 +0530
+++ b/drivers/i2c/busses/i2c-piix4.c	2014-05-06 22:12:26.333651900 +0530
@@ -295,7 +295,8 @@
 	/* Aux SMBus does not support IRQ information */
 	if (aux) {
 		dev_info(&PIIX4_dev->dev,
-			 "SMBus Host Controller at 0x%x\n", piix4_smba);
+			 "Auxiliary SMBus Host Controller at 0x%x\n",
+			 piix4_smba);
 		return piix4_smba;
 	}
 
